packages/relay/src/lib/eth.ts

```diff
@@ -64,42 +64,47 @@ export class EthImpl implements Eth {
    */
   private readonly mirrorNode: MirrorNode;
 
-    /**
+  /**
    * The interface through which we interact with the mirror node
    * @private
    */
-     private readonly mirrorNodeClient: MirrorNodeClient;
-
+  private readonly mirrorNodeClient: MirrorNodeClient;
 
   /**
    * The logger used for logging all output from this class.
    * @private
    */
-  private readonly logger:Logger;
+  private readonly logger: Logger;
 
   /**
    * The ID of the chain, as a hex string, as it would be returned in a JSON-RPC call.
    * @private
    */
-  private readonly chain:string;
+  private readonly chain: string;
 
   /**
    * Create a new Eth implementation.
    * @param clientMain
    * @param mirrorNode
    * @param logger
    */
-  constructor(clientMain:Client, mirrorNode:MirrorNode, mirrorNodeClient:MirrorNodeClient, logger:Logger) {
+  constructor(
+    clientMain: Client,
+    mirrorNode: MirrorNode,
+    mirrorNodeClient: MirrorNodeClient,
+    logger: Logger,
+    chain: string
+  ) {
     this.clientMain = clientMain;
     this.mirrorNode = mirrorNode;
     this.mirrorNodeClient = mirrorNodeClient;
     this.logger = logger;
 
     // Compute the chainId
     // FIXME: Should default to a "dev net" number. Is this it?
-    const configuredChainId = process.env.CHAIN_ID || '298';
-    this.chain = EthImpl.prepend0x(Number(configuredChainId).toString(16));
-    this.logger.info("Running with chainId=%s", this.chain);
+    //const configuredChainId = process.env.CHAIN_ID || '298';
```

Comment:
```suggestion
```

Reply:
Done.  Moved log line to where we calculate the chain id.